src/test/resources/log4j.properties

```diff
@@ -28,3 +28,16 @@ log4j.logger.org.eclipse.jetty=ERROR
 log4j.logger.org.glassfish.jersey.internal=ERROR
 
 log4j.logger.io.confluent.connect.elasticsearch=DEBUG
+
+
+# Send the Errant Reporter logs to a DLQ file, rolling the file connect-dlq.log based on a Max Size of 1MB.
+#
+log4j.appender.dlqLogAppender=org.apache.log4j.RollingFileAppender
```

Comment:
modifying log4j file isn't necessary

Reply:
This PR is primarily to test these changes https://github.com/confluentinc/ce-kafka/pull/2826 
Doesn't have to be merged here